src/components/Question.tsx

```diff
@@ -12,11 +13,13 @@ const Question = ({
   isCorrect: boolean;
 }) => {
   return (
-    <div className="qr-pb-4">
-      <p className="qr-pb-4 qr-text-lg">{text}</p>
+    <div className={styles.question}>
+      <p>{text}</p>
 
       {showResult ? (
-        <p>Your answer was {isCorrect ? ' correct ✅' : ' not correct ❌'}</p>
+        <p lassName={styles.p}>
```

Comment:
`<p className={styles.p}>`

Reply:
Thanks for spotting that! We need to run `tsc` at some stage to automatically detect such errors.